src/serlo-editor/editor-ui/plugin-toolbar/text-controls/utils/math.ts

```diff
@@ -42,7 +42,7 @@ export function toggleMath(editor: SlateEditor) {
         type: 'math',
         src: SlateEditor.string(editor, trimmedSelection as Location) || '',
         inline: true,
-        children: [],
+        children: [{ text: '' }],
```

Comment:
why is this change needed?

Reply:
As far as I understand, slate will place an empty text element there when running normalizations. It isn't need at all in this pull request. I should have left a comment here. We can remove it if you want.

Reply:
so you are saying that slate will include this anyway if we don't provide it?

Reply:
Yes! It is also one of slates "rules", see https://docs.slatejs.org/concepts/11-normalizing (first rule)

Reply:
nice, thanks for looking it up.